George/marketing.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Guidr</title>
+  <link href="css/marketing.css" rel="stylesheet">
+</head>
+
+<body>
+    
+    <header>
+        <nav class='Nav-Background'>
+            <img src="assets/Logo.svg" class="Logo">
+            
+        <div class='nav-btns'>
+            <div class='Button-BG btn-light'><span class='Button-BG-text'>REGISTER</span></div>
+            <div class='Button-BG btn-dark'><span class='Button-BG-text'>SIGN IN</span></div>
+        </div>
+        </nav>       
+    </header>
+    <div class='content'>
+        <div class='content-box'>
+            <h1>Guidr: Your Guide Portfolio Blah Blah Blah.</h1>
+            <div class='content-text'>
+                John was trying to find an easy way to log his guided trips. 
+                He loves hiking and makes a living guiding people on trips through the National Parks of Utah. 
+                He would try writing a log on paper but it would be messy and unorganized. 
+                It took too much time and he wasn’t keeping up with it. 
+                He found Guidr when browsing hiking logs online.             </div>
+        </div>
+
+    </div>
+
+    <footer class="footer">
+        <div class='Footer-Background'>
+            <div class='Footer-Content'>© 2019 Guidr</div>
+        </div>
+    </footer>
```

Comment:
Your footer could use a bit of margin on the top (for the sake of ensuring some space between it and the content div).